x-pack/plugins/apm/server/routes/alerts/rule_types/error_count/register_error_count_rule_type.test.ts

```diff
@@ -172,7 +172,7 @@ describe('Error count alert', () => {
       reason: 'Error count is 3 in the last 5 mins for bar. Alert when > 2.',
       threshold: 2,
       triggerValue: 3,
-      interval: '5m',
+      interval: '5 mins',
```

Comment:
I don't this is breaking but I saw @dgieselaar had a comment about something like this earlier.

Reply:
I think this is different than what @dgieselaar mentioned about, in this case, we already send `5m`, we're just changing it to `5 mins`. @dgieselaar do you think this would be a breaking change too?

Reply:
Technically, yes, but don't think it matters in practice unless people are parsing intervals.